ArduPlane/Attitude.cpp

```diff
@@ -360,7 +379,16 @@ void Plane::stabilize_yaw()
     /*
       now calculate rudder for the rudder
      */
-    const float rudder_output = calc_nav_yaw_coordinated();
+    float rudder_output = calc_nav_yaw_coordinated();
+
+#if AP_PLANE_SYSTEMID_ENABLED
+    if (control_mode->supports_fw_systemid()) {
+        Vector3f offset;
+        auto &systemid = plane.g2.systemid;
+        offset = systemid.get_output_offset();
+        rudder_output += offset.z * 4500.0f;
```

Comment:
for most users their rudder will already receive the aileron signal scaled by KFF_RDDRMIX (default 0.5). So this will mean we are doubling up that offset signal when doing systemID on the roll axis. I think until we think through a good plan for yaw that we not try to include rudder in systemID for fixed wing

Reply:
Not sure why this is an issue. It make yaw inputs directly to the servo. This would be one way of isolating the yaw response.  If you are firm on no including yaw at this point then I will remove it.

Reply:
Yaw and throttle inputs are removed.  We can address this again after this feature gets implemented.  but It will require another log message to collect the necessary data.